src/components/date-input/_macro.njk

```diff
@@ -69,7 +83,7 @@
         }) %}
             {{ fields | safe }}
         {% endcall %}
-    {% elif params.noWrap %}
+    {% elif params.noWrap or numberOfFields == 1 %}
```

Comment:
What happens if you include day, month and year so `numberOfFields == 3`?

Reply:
It will automatically wrap in a fieldset, this basically says don't wrap in a fieldset if there's only one field